### PR TITLE
This is handy for not comitting .vs profiles

### PR DESCRIPTION
**Pull request by fir3pho3nixx**

### CODE REVIEW

File: Changes.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - Fix unit test with weak reference broken by garbage collector changes in .NET 4.6.x (@ivan-danilov, #138)
 - Fix performance counter instances hanging around after the process using Windsor has ended (@mackenzieajudd, #146, #149)
 - Fix version of Castle.Core dependency in NuGet packages to indicate Castle.Core 4.0 is incompatible (#161)
+- Fix IL interpretation of Ldarg_N from LOCAL 0 to LOCAL [0,1] in OpCodes so test FluentRegistrationTestCase.Can_publish_events_via_AllTypes could pass on Windows 10 Home (build 14393.693) VS 2015 Update 3 using NET45-X (@fir3pho3nixx)
```
Comment (fir3pho3nixx) [inline Changes.txt]: This has been reverted

File: src/Castle.Facilities.EventWiring/OpCodeValues.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@ internal enum OpCodeValues : ushort
 	{
 		Nop = 0x0000,
 		Ldarg_0 = 0x0002,
+        Ldarg_1 = 0x0003,
```
Comment (fir3pho3nixx) [inline src/Castle.Facilities.EventWiring/OpCodeValues.cs]: This has been reverted

**Comment by fir3pho3nixx** [comment]
Think will recreate this PR so it does not have reverts in it.

**Comment by jberezanski** [comment]
No need to make a different PR. You could squash the two commits together using `git rebase -i` and force-push to github - the PR would update itself.

**Pull request closed without merging**